### PR TITLE
Get docs working with Docker

### PR DESCRIPTION
**Pull request by danieldraper**
## 💡 Why

Make it easier to get the docs running on Intel and M1 macs

## 📝 What

Get Docker updated and running with docker-compose

PS. Thanks Igor for fixing my mess ⭐

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,17 +7,32 @@ VOLUME /srv/slate/source
...
+
+RUN apt-get update && apt-get install -y --no-install-recommends gnupg
+ADD https://dl.yarnpkg.com/debian/pubkey.gpg /tmp/yarn-pubkey.gpg
+RUN apt-key add /tmp/yarn-pubkey.gpg && rm /tmp/yarn-pubkey.gpg
+RUN echo 'deb http://dl.yarnpkg.com/debian/ stable main' > /etc/apt/sources.list.d/yarn.list
 
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
         build-essential \
+        libpq-dev \
```
Comment (kieranjogrady) [inline Dockerfile]: Do we need to install `libpq-dev`?

**Review by kieranjogrady — APPROVED** [review]
Thanks for this Dan. I've left a note about a system dependency I suspect we don't need to install in the docker image.

**Review by chrisixion — APPROVED** [review]
Appears sensible captain 🧑‍✈️ (disclaimer: I'm neither on M1 nor working in Docker 😅 )

File: Dockerfile (hunk 1 above)
Comment (danieldraper) [reply Dockerfile]: Nope removed

**Review by markbiegel — APPROVED** [review]
Noice 👌

**Comment by corroded** [comment]
ditto with @chrisixion , I'm neither running M1 nor docker but these changes look sensible :ok_hand:

**Review by corroded — APPROVED** [review]

**Pull request merged**